server/views/components/promo/index.njk

```diff
@@ -3,64 +3,76 @@
 {#TODO Test with screen reader#}
 {#TODO jsConfig#}
 {#TODO className unitlity for promo classes#}
-<a href="{{ url }}" class="promo  {% if meta.type === 'gallery' or meta.type === 'podcast' or meta.type === 'video' %}promo--{{meta.type}}{% endif %} {% for modifier in modifiers %}promo--{{ modifier }} {% endfor %} ">
+
+<a href="{{ promo.url }}" class="promo {% for modifier in promo.modifiers %}promo--{{ modifier }} {% endfor %}">
   <div class="promo__image-container">
     <img class="promo__image"
-      src="{{ image.sources[(image.sources | length) - 1].src }}"
-        srcset="{%- for source in image.sources %}
-          {{ source.src }} {{ source.size }}w
-          {%- if (source !== (image.sources | last)) %}, {%- endif %}
-        {%- endfor %}"
-        alt="" />
+      src="{{ promo.article.mainImage.contentUrl }}"
+      alt="{{ promo.article.mainImage.caption }}" />
 
-    {% for modifier in modifiers %}
+    {% for modifier in promo.modifiers %}
       {% if modifier === 'series' %}
         <svg class="promo__defs">{# TODO proper filter - see Tomi; and only want this on the page once, where's best to move?#}
           <defs>
             <filter id="duotone" color-interpolation-filters="sRGB" x="0" y="0" height="100%" width="100%">
               <feColorMatrix type="matrix"
-                values="0.90 0 0 0   0.40 
-                        0.95 0 0 0  -0.10  
-                      -0.20 0 0 0   0.65 
+                values="0.90 0 0 0   0.40
+                        0.95 0 0 0  -0.10
+                      -0.20 0 0 0   0.65
                           0  0 0 1   0" />
             </filter>
           </defs>
         </svg>
-        <img class="promo__image-mask promo__clip-path--{{ [1,2,3,4,5,6,7,8,9,10] | random }}" src="{{image.sources[(image.sources | length) - 1].src}}" alt="" />
+        <img class="promo__image-mask promo__clip-path--{{ [1,2,3,4,5,6,7,8,9,10] | random }}" src="{{ promo.article.mainImage.contentUrl }}" alt="" />
```

Comment:
Possibly – it was for my own sanity trying to minimise the amount of things that were liable to break. Will see about getting `sources` back in.